calculator.js

```diff
@@ -1,5 +1,70 @@
-const exampleAdditionInput = {
+//check input is valid
+function number_validation(number) {
+  if (isNaN(number)) {
+    return false;
+  }
+  if (number == null || number === ' ') {
+    return false;
+  }
+  else {
+    return Number(number)
+  }
+}
+
+const addition = {
+  num1: 3,
+  num2: 5,
+  operation: ['add', '+'],
+}
+
+const subtraction = {
   num1: 3,
   num2: 5,
-  operation: 'add',
+  operation: ['subtract', '-'],
```

Comment:
Rather than making these arrays, maybe do it like this:

```javascript
const subtraction = {
  num1: 3,
  num2: 5,
  operation: 'subtract',
};

const otherSubtraction = {
  num1: 3,
  num2: 5,
  operation: '+',
};

```

This way you can simulate the user entering either option, rather than somehow simulate them entering both options.